packages/bento-design-system/src/DateField/DateField.tsx

```diff
@@ -109,17 +124,35 @@ function SingleDateField({ disabled, readOnly, ...props }: Extract<Props, { type
           shortcuts={shortcuts}
         />
       )}
+    </>
+  );
+
+  return props.isStandalone ? (
+    datePicker
+  ) : (
+    <Field
+      {...props}
+      disabled={disabled}
+      labelProps={labelProps}
+      assistiveTextProps={descriptionProps}
+      errorMessageProps={errorMessageProps}
+    >
+      {datePicker}
     </Field>
   );
 }
 
-function RangeDateField({ disabled, readOnly, ...props }: Extract<Props, { type: "range" }>) {
+function RangeDateField({
+  disabled,
+  readOnly,
+  ...props
+}: Extract<InternalDateProps, { type: "range" }>) {
   const localTimeZone = getLocalTimeZone();
   const internalProps = {
     ...props,
     isDisabled: disabled,
     isReadOnly: readOnly,
-    validationState: props.issues ? "invalid" : "valid",
+    validationState: !props.isStandalone && props.issues ? "invalid" : "valid",
```

Comment:
Is this correct? Is there a specific reason to ignore the `validationState` when used as a standalone component?

Reply:
I did it because`validationState` is relevant only for the _field_ part, the standalone component doesn't use it

https://github.com/buildo/bento-design-system/blob/c5f26ca666fcd5c1ce0625ce9a27525d0f36b6c5/packages/bento-design-system/src/DateField/Input.tsx#L31-L43

Should we also add it to the standalone component? It would be coherent with `NumberInput` actually

Reply:
🤔 I see the `internalProps` including it are passed only to `useDatePicker` and `useDatePickerState` which, according to the documentation, don't take a `validationState`. So I think it's irrelevant 👍